src/dialogs/PaymentErrorDialog.vue

```diff
@@ -6,7 +6,7 @@
       <v-card-text id="dialog-text">
         <!-- display common message -->
         <div class="font-15" v-if="!isRoleStaff">
-          <p>We are unable to process your payment at this time. This {{filingName}} has been saved
+          <p>We are unable to process your payment at this time. This Application has been saved
```

Comment:
Do you think it's a good idea to keep the prop here? I know it's only used once and it's in `App.vue` but maybe down the line we might use it? I'm asking since it's used a lot in Filings UI. Thoughts Sev?

Reply:
I can't decide.

It was originally coded with the prop so the filing name could be specified, but we only ever used the same name.

I'll simply revert this.

Reply:
Alright, sounds good 👍